src/Microsoft.TemplateEngine.Cli/PostActionProcessors/PostActionProcessorBase.cs

```diff
@@ -21,12 +24,19 @@ protected IReadOnlyList<string> GetTargetForSource(ICreationEffects2 creationEff
                 {
                     if (g.IsMatch(change.SourceRelativePath))
                     {
-                        results.Add(change.TargetRelativePath);
+                        results.Add(NormalizePath(outputBasePath, change.TargetRelativePath));
                     }
                 }
             }
-
             return results;
         }
+
+        /// <summary>
+        /// Gets normalized absolute path of <paramref name="basePath"/> + <paramref name="relativePath"/>.
+        /// </summary>
+        protected static string NormalizePath(string basePath, string relativePath)
```

Comment:
I'm not sure if `IsPathRooted` is what we need. We care that the path is absolute, https://docs.microsoft.com/en-us/dotnet/api/system.io.path.ispathfullyqualified?view=net-6.0 might be a better option. 
However, it might be just bad naming / description, `basePath` can be relative, but `Path.GetFullPath` will make it absolute anyway.

Reply:
I actually discovered https://docs.microsoft.com/en-us/dotnet/api/system.io.path.getfullpath?view=net-6.0#system-io-path-getfullpath(system-string-system-string) which does exactly what `NormalizePath` was supposed to do. It also throws in case `basePath` is not fully qualified path.